go/store/types/blob_test.go

```diff
@@ -273,6 +273,7 @@ func TestBlobConcat(t *testing.T) {
 
 	b7, b8 := split(b, 1e6/2)
 	b78, err := b7.Concat(context.Background(), b8)
+	assert.NoError(err)
```

Comment:
All of these asserts should be requires (so the test halts). Otherwise the following lines can die with nil pointer panics.

Reply:
I made this change throughout the package, save for functions where `testing.T` wasn't available.